toolkit/docs/formats/imageconfig.md

```diff
@@ -156,6 +156,50 @@ A sample KernelOptions specifying a default kernel and a specialized kernel for
 },
 ```
 
+### ReadOnlyVerityRoot
+"ReadOnlyVerityRoot" key controls making the root filesystem read-only using dm-verity.
+It will create a verity disk from the partition mounted at "/". The verity data is stored as
+part of the image's initramfs. More details can be found in [Misc: Read Only Roots](../how_it_works/5_misc.md#dm-verity-read-only-roots)
+
+#### Considerations
+Having a read-only root filesystem will change the behavior of the image in some fundamental ways. There are several areas that should be considered before enabling a read-only root:
+
+##### Writable Data
+Any writable data which needs to be preserved will need to be stored into a separate writable partition. The `TmpfsOverlays` key will create throw-away writable partitions which are reset on every boot. The example configs create an overlay on `/var`, but the more refined the overlays are, the more secure they will be.
+
+##### GPL Licensing
+If using a read-only root in conjunction with a verified boot flow that uses a signed initramfs carefully consider the implications on GPLv3 code. The read-only nature of the filesystem means a user cannot replace GPLv3 components without re-signing a new initramfs.
+
+##### Users
+Since users are controlled by files in `/etc` they are read-only when this is set. It is recommended to either use SSH key based login, or pre-hash the password to avoid storing passwords is plain text in the config files (See [Users](#users)).
+
+##### Separate `/boot` Partition
+Since the root partition's hash tree is stored as part of the initramfs, the initramfs cannot be stored on the same root partition (it would invalidate the measurements). To avoid this a separate `/boot` partition is needed to house the hash tree (via the initramfs).
+
+##### ISO
```

Comment:
Pending @niontive's changes.  Maybe comment this out for now?

Reply:
Was going to merge those changes in first, then do this PR. I'll add clarification that we currently only support the cmdline installer though.